server.js

```diff
@@ -1,35 +1,107 @@
+/* eslint-disable linebreak-style */
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import listEndpoints from 'express-list-endpoints'
+import dotenv from 'dotenv'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import exercisesData from './data/exercises.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+dotenv.config()
+
+const mongoUrl = process.env.MONGO_URL || 'mongodb://localhost/project-mongo'
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const exerciseSchema = new mongoose.Schema({
+  exerciseID: Number,
+  name: String,
+  category: String,
+  muscle_group: String,
+  target_muscle: String,
+  instructions: String,
+  img: String
+})
+
+const Exercise = mongoose.model('Exercise', exerciseSchema)
+
+if (process.env.RESET_DB) {
+  const seedDB = async () => {
+    await Exercise.deleteMany()
+
+    exercisesData.forEach((exercise) => {
+      const newExercises = new Exercise(exercise)
+      newExercises.save()
+    })
+  }
+  seedDB()
+}
+
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
-// Start defining your routes here
+// Endpoints
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
+})
+
+// All exercises
+app.get('/exercises', (req, res) => {
+  Exercise.find().then((exercises) => {
+    res.json({ data: exercises })
+  })
+})
+
+// Find exercise by id
+app.get('/exercises/:exerciseId', async (req, res) => {
+  const { exerciseId } = req.params
```

Comment:
Here I think it would be better to use the automatic id that is created in the database (the one that is called _id) since that is always guaranteed to be unique.